.github/workflows/SublimationBonjour.yml

```
@@ -3,112 +3,194 @@ on:
+    if: ${{ !contains(github.event.head_commit.message, 'ci skip') }}
     strategy:
+      fail-fast: false
       matrix:
-        image-name: ["swift:6.0-noble", "swiftlang/swift:nightly-6.1-jammy"]
-    container:
-      image: ${{ matrix.image-name }}
+        os: [noble, jammy]
+        swift:
+          - version: "6.0"
+          - version: "6.1"
+          - version: "6.1"
```

actionlint (1.7.7)

duplicate value {"nightly": "true", "version": "6.1"} is found in matrix "swift". the same value is at line:20,col:13

(matrix)